tests/mypy/test_mypy.py

```diff
@@ -105,6 +104,8 @@ def build(self) -> List[Union[Tuple[str, str], Any]]:
     # One-off cases
     + [
         ('mypy-plugin.ini', 'custom_constructor.py'),
+        ('mypy-plugin.ini', 'config_conditional_extra.py'),
+        ('mypy-plugin.ini', 'plugin_optional_inheritance.py'),
```

Comment:
I just moved this down here to reduce the number of mypy tests by 1, since I added this yesterday and forgot I could add it down here instead of above to reduce the number of runs.